exampleCourse/questions/element/details/question.html

```diff
@@ -0,0 +1,9 @@
+<pl-question-panel>
+  <pl-details title="Hint">
+    <p>For this question, you should click on the next header labelled "Large table of data" to view the content.</p>
+  </pl-details>
+
+  <pl-details title="Large table of data with show=false" show="false">
+    <pl-dataframe params-name="df"></pl-dataframe>
+  </pl-details>
```

Comment:
Most demo questions use some text in between instances to explain what's going on. This shows how the question is expected to be used.

Reply:
Added some more details in 83852b2af1d2fc5b2ee94a4dcf32b7f111cb37fb.